lib/model.js

```diff
@@ -3266,6 +3288,9 @@ class Model {
     if (updatedAtAttr && options.fields.length >= 1 && options.fields.indexOf(updatedAtAttr) === -1) {
       options.fields.push(updatedAtAttr);
     }
+    if (versionAttr && options.fields.length >= 1 && options.fields.indexOf(versionAttr) === -1) {
```

Comment:
`arr.includes()` is not available in Node 4, and we actually try to use less lodash